scripts/deploy.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/sh
+set -e
```

Comment:
just a question but what does `set -e` do here?

Reply:
if a command returns a non 0 exit code, it aborts the script with that exit code rather than continuing.

Shell defaults to "best effort", where it skips invalid commands.

Otherwise you'd have it where the shell would crash in interactive mode each time any command you run fails :P